checks/evaluation/signed_releases_test.go

```diff
@@ -15,96 +15,295 @@
 package evaluation
 
 import (
+	"fmt"
 	"testing"
 
-	"github.com/google/go-cmp/cmp"
-	"github.com/google/go-cmp/cmp/cmpopts"
-
 	"github.com/ossf/scorecard/v4/checker"
-	"github.com/ossf/scorecard/v4/clients"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/releasesAreSigned"
+	"github.com/ossf/scorecard/v4/probes/releasesHaveProvenance"
 	scut "github.com/ossf/scorecard/v4/utests"
 )
 
+const (
+	release0 = 0
+	release1 = 1
+	release2 = 2
+	release3 = 3
+	release4 = 4
+)
+
+const (
+	asset0 = 0
+	asset1 = 1
+	asset2 = 2
+	asset3 = 3
+	asset4 = 4
+	asset5 = 5
+	asset6 = 6
+	asset7 = 7
+	asset8 = 8
+	asset9 = 9
+)
+
+func negativeSignedProbe(totalReleases, release, asset int) finding.Finding {
+	return finding.Finding{
+		Probe:   "releasesAreSigned",
+		Outcome: finding.OutcomeNegative,
+		Values: map[string]int{
+			"totalReleases":                   totalReleases,
+			fmt.Sprintf("v%d", release):       int(releasesAreSigned.ValueTypeRelease),
+			fmt.Sprintf("artifact-%d", asset): int(releasesAreSigned.ValueTypeReleaseAsset),
+		},
+	}
+}
+
+func positiveSignedProbe(totalReleases, release, asset int) finding.Finding {
+	return finding.Finding{
+		Probe:   "releasesAreSigned",
+		Outcome: finding.OutcomePositive,
+		Values: map[string]int{
+			"totalReleases":                   totalReleases,
+			fmt.Sprintf("v%d", release):       int(releasesAreSigned.ValueTypeRelease),
+			fmt.Sprintf("artifact-%d", asset): int(releasesAreSigned.ValueTypeReleaseAsset),
+		},
+	}
+}
+
+func negativeProvenanceProbe(totalReleases, release, asset int) finding.Finding {
+	return finding.Finding{
+		Probe:   "releasesHaveProvenance",
+		Outcome: finding.OutcomeNegative,
+		Values: map[string]int{
+			"totalReleases":                   totalReleases,
+			fmt.Sprintf("v%d", release):       int(releasesHaveProvenance.ValueTypeRelease),
+			fmt.Sprintf("artifact-%d", asset): int(releasesHaveProvenance.ValueTypeReleaseAsset),
+		},
+	}
+}
+
+func positiveProvenanceProbe(totalReleases, release, asset int) finding.Finding {
```

Comment:
let's remove totalReleases from the function argument.

Reply:
https://github.com/ossf/scorecard/pull/3610/commits/0e418e78005b8838f55b67d7ca9da7a854f9521f